### PR TITLE
Surveyor login

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Adds support for creating surveyor accounts from /orgs/surveyor on mobile
* Moves manage users to its own page

### CODE REVIEW

File: temba/orgs/models.py [hunk 1]
```diff
@@ -208,6 +208,9 @@ class Org(SmartModel):
     brand = models.CharField(max_length=128, default=settings.DEFAULT_BRAND, verbose_name=_("Brand"),
                              help_text=_("The brand used in emails"))
 
+    surveyor_password = models.CharField(null=True, max_length=128, default=None,
+                                         help_text='A password that allows users to register as surveyors')
```
Comment (nicpottier) [inline temba/orgs/models.py]: localize

File: temba/orgs/tests.py [hunk 2]
```diff
@@ -645,6 +645,63 @@ def test_surveyor_invite(self):
...
+        self.org.save()
+
+        # now lets try again
+        post_data = dict(surveyor_password='nyaruka')
+        response = self.client.post(url, post_data)
+        self.assertContains(response, 'Enter your details below to create your account.')
+
+        # now try creating an account on the second step without and surveyor_password
+        post_data = dict(first_name='Marshawn', last_name='Lynch',
+                         password='beastmode24', email='beastmode@seahawks.com')
```
Comment (nicpottier) [inline temba/orgs/tests.py]: :)

File: temba/orgs/views.py [hunk 3]
```diff
@@ -890,6 +890,13 @@ def post(self, request, *args, **kwargs):
                 return HttpResponseRedirect(self.get_success_url())
             return super(OrgCRUDL.Update, self).post(request, *args, **kwargs)
 
+    class Accounts(InferOrgMixin, OrgPermsMixin, SmartUpdateView):
+        success_url = "@orgs.org_home"
+        success_message = ""
+        submit_button_name = _("Save Changes")
+        title = 'User Accounts'
+        fields = ('surveyor_password',)
```
Comment (nicpottier) [inline temba/orgs/views.py]: Don't we need to validate that `surveyor_password` is unique across orgs? Might not want to have an error message quite as specific as "this is already taken" though.

File: temba/orgs/views.py [hunk 4]
```diff
@@ -1272,10 +1278,126 @@ def get_context_data(self, **kwargs):
...
+                                       help_text=_("Your password, at least eight letters please"))
+
+            def __init__(self, *args, **kwargs):
+                super(OrgCRUDL.Surveyor.RegisterForm, self).__init__(*args, **kwargs)
+
+            def clean_email(self):
+                email = self.cleaned_data['email']
+                if email:
+                    if User.objects.filter(username__iexact=email):
+                        raise forms.ValidationError(_("That email address is already used"))
```
Comment (nicpottier) [inline temba/orgs/views.py]: Can be v1.1 but this seems like a likely path, we may want to redirect them to password reset once we have a better flow for that in surveyor.

**Comment by nicpottier** [comment]
looks good

File: temba/orgs/views.py (hunk 3 above)
Comment (ericnewcomer) [inline temba/orgs/views.py]: Doh! Yes indeed. Good catch.

**Pull request merged**